common.vars.pkr.hcl

```diff
@@ -1 +1 @@
-postgres-version = "15.1.1.95"
+postgres-version = "15.1.1.96"
```

Comment:
This feels wrong? It was in another file I think.

Reply:
@hf yes you got it. We changed it and now these versions live at https://github.com/supabase/postgres/pull/1389#discussion_r1896669812